### PR TITLE
docs(rich-text): remove beta

### PR DESCRIPTION
**Pull request by adnasa**
#### Summary

- removes `beta` flag from rich text #1737

### CODE REVIEW

**Review by tdeekens — COMMENTED** [review]
There is a pending update of the underlying library. Should we get that in before we get it out of beta? 🤔

**Comment by adnasa** [comment]
yeah let's try to update it then

File: .changeset/eleven-meals-remain.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+---
+'@commercetools-uikit/localized-rich-text-input': major
+'@commercetools-uikit/rich-text-input': major
```
Comment (emmenko) [inline .changeset/eleven-meals-remain.md:3]: Not sure we need to bump a major here, technically nothing changed 🤔

Comment (adnasa) [reply .changeset/eleven-meals-remain.md:3]: yeah good thinking. 
we can mark it as `patch` but is part of the v12 release anyway :)

Comment (adnasa) [reply .changeset/eleven-meals-remain.md]: https://github.com/commercetools/ui-kit/pull/1825/commits/f5fe8d457275c5d8b80ef8ba9d5ff002319d13d3

**Review by emmenko — APPROVED** [review]

**Pull request merged**